tests/system/providers/google/cloud/dataprep/example_dataprep.py

```diff
@@ -0,0 +1,175 @@
+    },
+)
+
+with models.DAG(
+    DAG_ID,
+    schedule_interval="@once",
+    start_date=datetime(2021, 1, 1),  # Override to match your needs
+    catchup=False,
+    tags=['example', 'dataprep'],
+    render_template_as_native_obj=True,
+) as dag:
+    # [START how_to_gcs_create_bucket_operator]
+    create_bucket_task = GCSCreateBucketOperator(
+        task_id="create_bucket",
+        bucket_name=GCS_BUCKET_NAME,
+        project_id=GCP_PROJECT_ID,
+    )
+    # [END how_to_gcs_create_bucket_operator]
+
+    # [START how_to_dataprep_run_job_group_operator]
+    run_job_group_task = DataprepRunJobGroupOperator(
+        task_id='run_job_group',
+        project_id=GCP_PROJECT_ID,
+        body_request={
+            "wrangledDataset": {"id": RECIPE_ID},
+            "overrides": WRITE_SETTINGS,
+        },
+    )
+    # [END how_to_dataprep_run_job_group_operator]
+
+    # [START how_to_dataprep_copy_flow_operator]
+    copy_task = DataprepCopyFlowOperator(
+        task_id="copy_flow",
+        project_id=GCP_PROJECT_ID,
+        flow_id=FLOW_ID,
+        name=f'dataprep_example_flow_{DAG_ID}_{ENV_ID}',
+    )
+    # [END how_to_dataprep_copy_flow_operator]
+
+    # [START how_to_dataprep_dataprep_run_flow_operator]
+    run_flow_task = DataprepRunFlowOperator(
+        task_id="run_flow",
+        project_id=GCP_PROJECT_ID,
+        flow_id="{{ task_instance.xcom_pull('copy_flow')['id'] }}",
+        body_request={
+            "overrides": {
+                RECIPE_NAME: WRITE_SETTINGS,
+            },
+        },
+    )
+    # [END how_to_dataprep_dataprep_run_flow_operator]
+
+    # [START how_to_dataprep_get_job_group_operator]
+    get_job_group_task = DataprepGetJobGroupOperator(
+        task_id='get_job_group',
+        project_id=GCP_PROJECT_ID,
+        job_group_id="{{ task_instance.xcom_pull('run_flow')['data'][0]['id'] }}",
+        embed="",
+        include_deleted=False,
+    )
+
+    # [START how_to_dataprep_get_jobs_for_job_group_operator]
+    get_jobs_for_job_group_task = DataprepGetJobsForJobGroupOperator(
+        task_id="get_jobs_for_job_group",
+        job_group_id="{{ task_instance.xcom_pull('run_flow')['data'][0]['id'] }}",
+    )
+    # [END how_to_dataprep_get_jobs_for_job_group_operator]
+
+    # [START how_to_dataprep_job_group_finished_sensor]
+    check_flow_status_sensor = DataprepJobGroupIsFinishedSensor(
+        task_id="check_flow_status",
+        job_group_id="{{ task_instance.xcom_pull('run_flow')['data'][0]['id'] }}",
+    )
+    # [END how_to_dataprep_job_group_finished_sensor]
+
+    # [START how_to_dataprep_job_group_finished_sensor]
+    check_job_group_status_sensor = DataprepJobGroupIsFinishedSensor(
+        task_id="check_job_group_status",
+        job_group_id="{{ task_instance.xcom_pull('run_job_group')['id'] }}",
+    )
+    # [END how_to_dataprep_job_group_finished_sensor]
+
+    # [START how_to_dataprep_delete_flow_operator]
+    delete_flow_task = DataprepDeleteFlowOperator(
```

Comment:
If the task persists and disappears only when removing (like gcs bucket) it should have a trigger rule all done to ensure it's actually deleted even if the test stops in the middle.
And the trigger should be "outside" documentation markers if there are any:
```
    # [START how_to_dataprep_delete_flow_operator]
    delete_flow_task = DataprepDeleteFlowOperator(
        task_id="delete_flow",
        flow_id="{{ task_instance.xcom_pull('copy_flow')['id'] }}",
    )
    # [END how_to_dataprep_delete_flow_operator]
    delete_flow_task.trigger_rule = TriggerRule.ALL_DONE

```

Reply:
Add trigger rule as suggested